CHANGELOG.md

```diff
@@ -2,12 +2,19 @@
 
 [![Nuget](https://img.shields.io/nuget/v/Microsoft.Azure.IoT.Edge.Module.svg)](https://www.nuget.org/packages/Microsoft.Azure.IoT.Edge.Module/)
 
+### 3.1.0-alpha (2019-12-24)
+* [Added] Add arm64v8.debug dockerfile
+
 ### 2.7.0 (2019-12-20)
 * [Updated] Pull base image from Microsoft Container Registry
 
 ### 2.6.0 (2019-11-01)
 * [Added] Add .dockerignore to ignore bin and obj folder
 
+### 3.0.0-alpha (2019-07-17)
```

Comment:
this version is not released right?

Reply:
The 3.0.0-alpha and 3.1.0-alpha has been pushed to Nuget as pre-release version. So I listed here.

Reply:
now we need remove the alpha right?

Reply:
Nuget doesn't support removing packages since this may break existing projects. We can unlist the pre-release version if necessary. Unlist can hide a version in the version list.
Do we need to unlist the pre-release version?

Reply:
Removed the changelog for pre-release version